tests/test_data/lcmodel/liver.COORD

```diff
@@ -19,7 +19,7 @@
    5 lines in following misc. output table
   FWHM = 0.297 ppm    S/N =  24                                            
   Data shift =-0.019 ppm                                                   
-  Ph:  36 deg       0.3 deg/ppm                                            
+  Ph:-136 deg       0.3 deg/ppm                                            
```

Comment:
This .COORD file is only used in one test. I modified it to include the edge case that was crashing previously.